shared/versions/ece/master.asciidoc

```diff
@@ -1,3 +1,3 @@
-:ece-version:  3.7.1
-:ece-version-short:  3.7
-:ece-version-link: 3.7
\ No newline at end of file
+:ece-version:  3.8.-1
```

Comment:
```suggestion
:ece-version:  3.8.1
```
I don't think that hyphen should be there.

Reply:
Yeah I was wondering why there was an hyphen in the first place!